### PR TITLE
New package: python3 tasklib

### PR DESCRIPTION
**Pull request by Pinoinha**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/vit/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+short_desc="VIT is a lightweight, fast, curses-based front end to Taskwarrior "
+maintainer="pinoinha <pinoinha@disroot.org"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+changelog="https://raw.githubusercontent.com/vit-project/vit/2.x/CHANGES.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=56676651164fb71878a0c1e4bb369cfa336d46e6fd0de7e6167b57fca7eaf662
+
+pre_build() {
+	pip3 install tasklib
```
Comment (ericonr) [inline srcpkgs/vit/template:18]: Add `python3-tasklib` to `hostmakedepends`.

**Comment by ericonr** [comment]
Actually, this should be together with the `vit` PR.

**Comment by Pinoinha** [comment]
> Actually, this should be together with the `vit` PR.

thank you for the warning. I'm new to contributing, so am still learning the "etiquette"

will close this PR in favor of the vit PR

File: srcpkgs/vit/template (hunk 1 above)
Comment (Pinoinha) [reply srcpkgs/vit/template:18]: just as a clarification: is this line needed? in the vit PR a fellow contributor said it shouldn't be essential

also the template already installs the files in the system

**Pull request closed without merging**